src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -2345,6 +2345,7 @@ private void model_ComputeModelDeserialized()
         {
             try
             {
+                filePath = Model.CurrentWorkspace.FileName;
```

Comment:
`filepath` is assigned but never used?

Reply:
It is used in the next line to read the text from this new current workspace.